DatadogCore/Tests/Matchers/RUMSessionMatcher.swift

```diff
@@ -74,18 +88,27 @@ internal class RUMSessionMatcher {
         fileprivate(set) var longTaskEvents: [RUMLongTaskEvent] = []
     }
 
+    /// RUM application ID for this session.
+    let applicationID: String
+    /// The ID of this session in RUM.
+    let sessionID: String
+
+    let applicationLaunchView: ViewVisit?
```

Comment:
could have tiny description

Reply:
I'll keep it this way - it's a move, not addition. And IDK what was the reason for this prop.